src/services/ProjectTokenHistoricalPriceService.ts

```diff
@@ -237,17 +237,19 @@ export class ProjectTokenHistoricalPriceService {
     const poolPTBalance = this.numberService.fromString(fromWei(lbpMgr.lbp.vault.projectTokenBalance, lbpMgr.projectTokenInfo.decimals));
     const poolFTBalance = this.numberService.fromString(fromWei(lbpMgr.lbp.vault.fundingTokenBalance, lbpMgr.fundingTokenInfo.decimals));
     const projectTokenAmount = this.numberService.fromString(lastSwap.tokenAmountOut);
-    // const fundingTokenAmount = this.numberService.fromString(lastSwap.tokenAmountIn);
     const forecastData = await this.priceService.getInterpolatedPriceDataPoints(
       poolPTBalance,
-      poolFTBalance,
+      poolFTBalance * (1 + lbpMgr.swapFeePercentage),
```

Comment:
I don't think this is necessary. When we get the `poolFTBalance` from the vault, my understanding is that all swap fees that had been collected so far are included in that `poolFTBalance`.

Reply:
You are correct @fabianschu

Reply:
removed the fee from pool balance